### ISSUE TITLE
Android: Starting with version 6.0.0-beta.6 - Backdrop Filter blur not working anymore

### ISSUE THREAD

**Issue by AlexDochioiu** [pichillilorenzo/flutter_inappwebview#1415]
- [x] I have read the [Get Started - Installation](https://inappwebview.dev/docs/get-started/installation/) section
- [x] I have read and done the [Get Started - Setup Android](https://inappwebview.dev/docs/get-started/setup-android/) section
- [x] I have read and done the [Get Started - Setup iOS](https://inappwebview.dev/docs/get-started/setup-ios/) section
- [x] I have already searched for the same problem

## Environment

| Technology           | Version       |
| -------------------- | ------------- |
| Flutter version      |   3.3.7       |
| Plugin version       | 6.0.0-beta.6  |
| Android version      |Android 12 (API 31)|
| iOS version          |   not relevant       |
| Xcode version        |    not relevant  |

Device information: Xiaomi Redmin Note 11 Pro

## Description

Starting with version `6.0.0-beta.6` up to `6.0.0-beta.12` ( **`6.0.0-beta.5` works well**), there seem to be issues drawing a blur (BackdropFilter) on Android when a WebView is present. Somehow the blur doesn't apply at all (outside of the webview as well) starting with that version.

**Expected behavior:** Blur over the whole screen should still work

**Current behavior:** There is no blur at all anywhere on the screen starting with the mentioned version

## Steps to reproduce

My exact scenario, is drawing a bottom sheet with a blurred overlay. What I am using is:

```yaml
modal_bottom_sheet: ^2.1.2
```

Code to draw the bottom sheet with a blurred barrier:

```dart
import 'dart:ui';

import 'package:flutter/material.dart';
import 'package:modal_bottom_sheet/modal_bottom_sheet.dart';

Future<T?> showBlurredBarBottomSheet<T>({
  required BuildContext context,
  required WidgetBuilder builder,
  double elevation = 2,
  ShapeBorder shape = const RoundedRectangleBorder(
    borderRadius: BorderRadius.only(topLeft: Radius.circular(25), topRight: Radius.circular(25)),
  ),
  Clip? clipBehavior,
  Curve? animationCurve,
  Duration? duration,
  RouteSettings? settings,
  double? closeProgressThreshold,
  AnimationController? secondAnimation,
  Color? backgroundColor,
  Color? barrierColor,
  bool bounce = false,
  bool useRootNavigator = true,
  bool isDismissible = true,
  bool enableDrag = true,
}) {
  final sheetContent = Column(
    mainAxisSize: MainAxisSize.min,
    crossAxisAlignment: CrossAxisAlignment.center,
    children: [
      const SizedBox(height: 15),
      Container(
        height: 3,
        width: 40,
        decoration: const BoxDecoration(
          color: Color(0xFF979797),
          borderRadius: BorderRadius.all(Radius.circular(6)),
        ),
      ),
      const SizedBox(height: 15),
      Flexible(
        flex: 1,
        fit: FlexFit.loose,
        child: SizedBox(width: double.infinity, child: builder(context)),
      ),
    ],
  );

  return showCustomModalBottomSheet(
    context: context,
    containerWidget: ((context, animation, child) => child),
    builder: (context) => BackdropFilter(
      filter: ImageFilter.blur(sigmaX: 5, sigmaY: 5),
      child: SafeArea(
        bottom: false,
        child: Material(
          shape: shape,
          clipBehavior: clipBehavior ?? (Theme.of(context).brightness == Brightness.dark ? Clip.antiAlias : Clip.none),
          color: backgroundColor ??
              (Theme.of(context).brightness == Brightness.dark ? const Color(0xB2191F2C) : const Color(0xFFFFFFFF)),
          elevation: elevation,
          child: Theme.of(context).brightness == Brightness.dark
              ? BackdropFilter(
                  // extra blur only on dark mode
                  filter: ImageFilter.blur(sigmaX: 10, sigmaY: 10),
                  child: sheetContent,
                )
              : Container(
                  decoration: const BoxDecoration(
                    color: Color(0xFFFFFFFF),
                    borderRadius: BorderRadius.vertical(top: Radius.circular(25)),
                    boxShadow: [BoxShadow(color: Colors.black26, blurRadius: 5, blurStyle: BlurStyle.outer)],
                  ),
                  child: sheetContent,
                ),
        ),
      ),
    ),
    elevation: elevation,
    shape: shape,
    clipBehavior: clipBehavior,
    animationCurve: animationCurve,
    duration: duration,
    settings: settings,
    closeProgressThreshold: closeProgressThreshold,
    secondAnimation: secondAnimation,
    barrierColor: barrierColor ??
        (Theme.of(context).brightness == Brightness.dark ? const Color(0x80000000) : const Color(0x147E7E33)),
    backgroundColor: Colors.transparent,
    bounce: bounce,
    expand: false,
    useRootNavigator: useRootNavigator,
    isDismissible: isDismissible,
    enableDrag: enableDrag,
  );
}
```

In app web view code with the settings:

```dart
InAppWebView(
  initialSettings: InAppWebViewSettings(
    transparentBackground: true, // to avoid white flash on dark mode
    javaScriptEnabled: true,
    // useShouldOverrideUrlLoading: true,
    mediaPlaybackRequiresUserGesture: false,
    useHybridComposition: true,
    allowsInlineMediaPlayback: true,
    rendererPriorityPolicy: RendererPriorityPolicy(
      rendererRequestedPriority: RendererPriority.RENDERER_PRIORITY_IMPORTANT,
      waivedWhenNotVisible: true,
      ),
    geolocationEnabled: false,
  ),
  onProgressChanged: (controller, progress) =>
    ref.read(webViewPageNotifierProvider.notifier).onProgressChanged(progress / 100.0),
  onLoadStart: (controller, url) =>
    ref.read(webViewPageNotifierProvider.notifier).onLoadStart(url?.toString()),
  onUpdateVisitedHistory: (controller, url, isReload) =>
    ref.read(webViewPageNotifierProvider.notifier).onUpdateVisitedHistory(url?.toString()),
  onWebViewCreated: (controller) =>
    ref.read(webViewPageNotifierProvider.notifier).onWebViewCreated(controller, context),
  onLoadStop: (controller, url) =>
    ref.read(webViewPageNotifierProvider.notifier).onLoadStop(url?.toString()),
);
```

## Working version: 6.0.0-beta.5

![6.0.0-beta.5](https://user-images.githubusercontent.com/38853913/200136457-86b6fce8-9015-4942-9b17-192995b92319.png)

## Not working version: 6.0.0-beta.6

![not_working](https://user-images.githubusercontent.com/38853913/200136537-6067a4a1-5ebb-4548-9672-2c7da7ce0581.png)

**Comment by AlexDochioiu** [comment]
@pichillilorenzo Seems that the issue is related to the hybrid composition changes introduced in beta.6. I expect it's got to do with hybridComposition using `initExpensiveAndroidView` starting with that version.

**Comment by AlexDochioiu** [comment]
P.S. disabling hybrid composition makes the backdrop filter work again starting with `beta.6`.

**Comment by pichillilorenzo** [comment]
@AlexDochioiu thanks, so the problem is related to how Flutter Hybrid Composition currently works.
Unfortunately, that's a bug of Flutter itself and not of this plugin.
I think you should open an issue on the official Flutter repository related to this bug.

So, currently, if the backdrop filter is a MUST for you, you should disable hybrid composition.

**Comment by sinahamedi67** [comment]
I had the same problem but resolved with [this commit](https://github.com/flutter/engine/pull/39244) on flutter engine which put the platform views behind the backdrop and released with Flutter 3.10 on stable channel.

**Comment by shubham16g** [comment]
Same here, I am developing a windows 11 clone. Lots of UI components have blur backdrop filter, but when InAppWebView appears on screen, all the backdrop filter blur disappears.

![image](https://github.com/user-attachments/assets/ce4d3e4c-720b-408a-bd62-bf81c7b61731)
![image](https://github.com/user-attachments/assets/028578e6-a09b-4598-a507-14524ad5e837)
